src/vos/vos_io.c

```diff
@@ -1564,10 +1564,10 @@ dkey_update_begin(struct vos_io_context *ioc)
 }
 
 int
-vos_publish_scm(struct vos_container *cont, struct vos_rsrvd_scm *rsrvd_scm,
-		bool publish)
+vos_publish_scm(struct vos_container *cont, void *data, bool publish)
 {
-	int	rc = 0;
+	struct vos_rsrvd_scm	*rsrvd_scm = data;
```

Comment:
I don't quite follow this change.

Reply:
That allows we use the buffer that is attached to DTX handle as the input parameter.